### DISCUSSION TITLE
Anyone willing to build me one?

### DISCUSSION THREAD

**Discussion by DThumbs** [alf45tar/PedalinoMini#289] [Q&A] [answered]
Hi,
I am pretty overwhelmed thinking I can make one of these, and I respect those that understand all of this!
Is anyone selling (or willing to build) a completed unit (and offer some basic support?)
Musically,
Dan

**Comment by DWSQUIRES** [comment]
Hey Dan, Dan here.
If you're in New Zealand I'd be happy to do the coding upload portion for you and send the board's out to you if that's the bit you find daunting (it's not too hard), else I can just send you a list of parts I bought off AliExpress and you could start there. I used a standard esp32 board and a separate display module. Still assembling a 6 switch version with external expression so it's fresh (ish) in the mind if you have any questions

**Reply by DThumbs** [reply]
Dan,
I'm in the US near Chicago.  if you don't mind sharing some links, I'll take a look at the parts?
I appreciate your reply!!

**Comment by alf45tar** [comment, marked as answer]
PedalinoMini is for makers and DIY lovers. It is not a ready from the shelf product.
 The fun is into learning something.

**Reply by DThumbs** [reply]
I do completely agree.  I WANT to learn this, but the curve is longer than my actual need timeline, and I am performing often.

**Comment by rigr** [comment]
It's not only the build, setting it up to your needs might easily take the
same amount of time. :)
